### PR TITLE
Diodexcelerate event

### PR DESCRIPTION
**Pull request by NikhilKottoli**
# Description

The homepage of Diodexcelerate, an event of ieee diode. 

Fixes # (issue)

## Dependencies

No new dependency 

## Type of Change

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration. (for bug fix / breaking change)
_Put an `x` in the boxes that apply_

- [x] Passed the run pre-commit
- [ ] Test B

---

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by yukitya-1811** [comment]
A few changes -
1. Change it to DiodeXcelerate (Capitalize the D)
2. Create a `base.html` file in the diodexcelerate templates folder. It's throwing Template Does Not Exist right now

**Comment by yukitya-1811** [comment]
Also change the colour of the text for the cards that display Embedathon, Elektrika etc. Keep it more suited to the aesthetics of the whole website so go for cooler colours like blue or teal. Also change the background of those cards. Ideally make them transparent so that actual background of the page is shown

File: corpus/templates/components/navbar_list.html [hunk 1]
```diff
@@ -92,6 +92,11 @@
                     Robotrix
                 </a>
             </li>
+            <li>
+                <a href="{% url 'diodexcelerate_home' %}">
+                    diodeXcelerate
+                </a>
+            </li>
```
Comment (imApoorva36) [inline corpus/templates/components/navbar_list.html:99]: move this to the top of the list

**Review by imApoorva36 — CHANGES_REQUESTED** [review]
nitpick

File: corpus/templates/components/navbar_list.html (hunk 1 above)
Comment (NikhilKottoli) [reply corpus/templates/components/navbar_list.html]: Changed it

**Review by imApoorva36 — APPROVED** [review]

**Review by yukitya-1811 — APPROVED** [review]

**Pull request merged**